src/lib/toolbar.ts

```diff
@@ -266,10 +266,11 @@ export const toolbarConfig: Record<ToolbarModeName, ToolbarMode> = {
     items: [
       {
         id: 'sketch-exit',
-        onClick: ({ modelingSend }) =>
+        onClick: ({ modelingSend }) => {
```

Comment:
If you want a lint rule or prettier config for this (all functions must use curly braces) I wouldn't be opposed, it's not the first time it's tripped one of us up. Last time it was an `if` statement with no braces and an early return that wrapped to the next line.

Reply:
I can revert this. I think I had some debug code and deleted it but forgot to remove the syntax. I am fine with our current pattern.